src/js/util.js

```diff
@@ -614,7 +614,7 @@ var Util;
         },
 
         isMediumEditorElement: function (element) {
-            return element && element.nodeType !== 3 && !!element.getAttribute('data-medium-editor-element');
+            return element && element.getAttribute && !!element.getAttribute('data-medium-editor-element');
```

Comment:
I had to add this line because apparently `.getAttribute` does not exist on `document`, and so this would throw an error if you passed in the document.  Who knew?